integrations/dd-sdk-android-okhttp/src/main/kotlin/com/datadog/android/okhttp/DatadogInterceptor.kt

```diff
@@ -481,6 +487,10 @@ internal constructor(
 
     internal companion object {
 
+        internal val STREAM_CONTENT_TYPES = arrayOf(
+            "text/event-stream"
```

Comment:
should we maybe try to include other stream types here? like for websocket, gRPC

Reply:
Good idea